caffe2/operators/hip/conv_transpose_op_miopen.hip

```diff
@@ -105,28 +105,28 @@ class MIOPENConvTransposeGradientOp final : public MIOPENConvTransposeOpBase {
             OperatorBase::GetSingleArgument<bool>("bestAlgoFound", false)),
         bestWeightAlgoFound_(
             OperatorBase::GetSingleArgument<bool>("bestAlgoFound", false)),
+        fwdAlgo_(miopenConvolutionFwdAlgoGEMM),
```

Comment:
Here `fwdAlgo_` is initialized before `bwdWeiAlgo_` while in line 140 and 141 they are declared in the reversed order

Reply:
Fixed in lines 140 and 141.